src/main/java/strategy/impl/BalanceOperation.java

```diff
@@ -0,0 +1,10 @@
+package strategy.impl;
+
+import strategy.OperationHandler;
+
+public class BalanceOperation implements OperationHandler {
+    @Override
+    public int apply(int quantity) {
```

Comment:
The apply method should check the result balance before saving it in the Storage to ensure it is positive. If the balance is negative, throw a RuntimeException.

Reply:
fix comment